pkg/exporter/process.go

```diff
@@ -53,7 +53,8 @@ type ExportingProcess struct {
 }
 
 type ExporterInput struct {
-	CollectorAddr       net.Addr
+	CollectorHostPort   string
+	CollectorTransport  string
```

Comment:
Same as above.
out-of-curiosity, why do we change the name to Transport instead of Protocol?

Reply:
Changed to Protocol to be consistent.